SED plugin update documentation (#863)
rapid7/insightconnect-plugins: sed/help.md

##### Input

> It accepts an expressions, options and a file input. The command-line options can be passed in via the options input.

Reviewer: ```suggestion
It accepts sed expressionss, options, and a file input. The command-line options can be passed in via the `options` input.
```

> It accepts sed expressionss, options, and a file input. The command-line options can be passed in via the `options` input.

Reviewer: ```suggestion
It accepts sed expressions, options, and a file input. The command-line options can be passed in via the `options` input.
```

|Name|Type|Default|Required|Description|Enum|Example|
|----|----|-------|--------|-----------|----|-------|
|bytes|bytes|None|True|File/bytes to Process|None|b25lIHR3byB0aHJlZSBmb3VyIGZpdmUgb25lIHR3bw==|
|expression|[]string|None|True|Sed Expression|None|["s/one/ONE/", "s/two/2/g"]|
|options|string|None|False|Sed Options|None|-r|

The `expression` input may contain a single expression, e.g `["s/one/ONE/"]` or multiply expressions, e.g `["s/one/ONE/","s/two/TWO/"]`.
The above example expression `["s/one/ONE/"]` will replace the first occurrence of a lowercase `one` with a capitalized `ONE` in each line of the provided file/bytes to process.

Example input:

```
{
  "bytes": "b25lIHR3byB0aHJlZSBmb3VyIGZpdmUgb25lIHR3bw==",
  "expression": [
    "s/one/ONE/",
    "s/two/2/g"
  ],
  "options": "-r"
}
```

##### Output

|Name|Type|Required|Description|
|----|----|--------|-----------|
|output|string|False|Processed String|

Example output:

```
{
  "output": "T05FIDIgdGhyZWUgZm91ciBmaXZlIG9uZSAy"
}
```

#### Process String

This action is used to process a string through the stream editor.

##### Input

> It accepts an expressions, options, and a string to process. The command-line options can be passed in via the options input.

Reviewer: ```suggestion
It accepts sed expressions, options, and a string to process. The command-line options can be passed in via the options input.
```

# Version History

> * 2.0.3 - Add input examples to documentation

Reviewer: ```suggestion
* 2.0.3 - Add action input and output examples to the documentation
```

> * 2.0.3 - Add the input examples in plugin spec | Update documentation

Reviewer: ```suggestion
* 2.0.3 - Add input examples to documentation
```
